llvm/lib/MC/DXContainerPSVInfo.cpp

```diff
@@ -162,6 +97,94 @@ void PSVRuntimeInfo::write(raw_ostream &OS, uint32_t Version) const {
                                llvm::endianness::little);
 }
 
+static constexpr size_t npos = StringRef::npos;
```

Comment:
Did you change these functions? They're moved from above which makes the diff look a lot larger than probably required. Can you put them back where they were originally to reduce the diff size?

Reply:
Yes!  I can fix this up!